nautobot/docs/models/dcim/platform.md

```diff
@@ -4,6 +4,11 @@ A platform defines the type of software running on a device or virtual machine.
 
 Platforms may optionally be limited by manufacturer: If a platform is assigned to a particular manufacturer, it can only be assigned to devices with a type belonging to that manufacturer.
 
-The platform model is also used to indicate which [NAPALM](https://github.com/napalm-automation/napalm/) driver and any associated arguments Nautobot should use when connecting to a remote device. The name of the driver along with optional parameters are stored with the platform.
+The platform model can be used to indicate which "network driver" Nautobot (and Jobs, Apps, etc.) should use when connecting to a remote device running this platform. This is a generic string corresponding loosely to a [Netmiko](https://github.com/ktbyers/netmiko) driver name. As there are many different libraries and applications for connecting to a device, rather than having a separate model field for each such connection type, Nautobot uses [netutils](https://netutils.readthedocs.io/en/latest/) to translate the generic network driver string into a variety of library-specific driver strings (Ansible "collection name", PyATS "OS" value, Scrapli "platform", etc.) which can be accessed via the UI, REST API, and GraphQL as needed. An administrator can extend or override the default translations provided by netutils by configuring the [`NETWORK_DRIVERS`](../../configuration/optional-settings.md#network_drivers) dictionary appropriately.
```

Comment:
```suggestion
The platform model can be used to indicate which "network driver" Nautobot (and Jobs, Apps, etc.) should use when connecting to a remote device running this platform. This is a generic string corresponding loosely to a [Netmiko](https://github.com/ktbyers/netmiko) driver name. As there are many different libraries and applications for connecting to a device, rather than having a separate model field for each such connection type, Nautobot uses [netutils](https://netutils.readthedocs.io/en/latest/) to translate the generic network driver string into a variety of library-specific driver strings (Ansible "collection name", PyATS "OS" value, Scrapli "platform", etc.) which can be accessed via the UI, REST API, and GraphQL as needed. An administrator can extend or override the default translations provided by netutils by configuring the [`NETWORK_DRIVERS`](../../configuration/optional-settings.md#network_drivers) dictionary appropriately or ideally creating a PR into [netutils](https://github.com/networktocode/netutils) to update the package.
```

Reply:
```suggestion
The platform model can be used to indicate which "network driver" Nautobot (and Jobs, Apps, etc.) should use when connecting to a remote device running this platform. This is a generic string corresponding loosely to a [Netmiko](https://github.com/ktbyers/netmiko) driver name. As there are many different libraries and applications for connecting to a device, rather than having a separate model field for each such connection type, Nautobot uses [netutils](https://netutils.readthedocs.io/en/latest/) to translate the generic network driver string into a variety of library-specific driver strings (Ansible "collection name", PyATS "OS" value, Scrapli "platform", etc.) which can be accessed via the UI, REST API, and GraphQL as needed. An administrator can extend or override the default translations provided by netutils by configuring the [`NETWORK_DRIVERS`](../../configuration/optional-settings.md#network_drivers) dictionary appropriately. (If your extensions are generally applicable, please consider making a pull request against [netutils](https://github.com/networktocode/netutils) to update the package!)
```